app/assets/javascripts/bastion/components/bst-global-notification.directive.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @ngdoc directive
+ * @name Bastion.components.directive:bstGlobalNotification
+ *
+ * @requires GlobalNotification
+ * @requires $scope
```

Comment:
Really a nitpick but we should list these in the same order as the parameters to the function.